src/olympia/discovery/tests/test_admin.py

```diff
@@ -1076,7 +1079,7 @@ def test_can_edit_with_discovery_edit_permission(self):
         item = Shelf.objects.create(
             title='Recommended extensions',
             shelf_type='extension',
-            criteria='/this/is/the/criteria',
```

Comment:
I feel like there should be too. I made these changes because with the `validate_criteria` validator, `test_can_edit_with_discovery_edit_permission` and `test_can_add_with_discovery_edit_permission` do not pass with just a placeholder in `criteria`. It led to the error message above about the call not matching a registered mock.